### PR TITLE
[Az.Accounts] Enable telemetry collection for several commands

### PR DESCRIPTION
**Pull request by NoriZC**
## Description

Some commands in Az.Accounts are lacking telemetrys because they overrided `beginProcessing()` incorrectly.
Upated commands in this PR Listed below:
- Disable-AzDataCollection
- Enable-AzDataCollection
- Remove-AzEnvironment
- Get-AzEnvironment
- Set-AzEnvironment
- Add-AzEnvironment<html>
- Import-AzContext

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Review by BethanyZhou — CHANGES_REQUESTED** [review]
Change log is required

File: src/Accounts/Accounts/Context/GetAzureRMContext.cs [hunk 1]
```diff
@@ -69,9 +69,9 @@ protected override IAzureContext DefaultContext
         [Parameter(Mandatory = false, ParameterSetName = ListAllParameterSet, HelpMessage = "Refresh contexts from token cache")]
         public SwitchParameter RefreshContextFromTokenCache { get; set; }
 
-        protected override void BeginProcessing()
+        protected override bool RequireDefaultContext()
```
Comment (BethanyZhou) [inline src/Accounts/Accounts/Context/GetAzureRMContext.cs:72]: why we need RequireDefaultContext()

Comment (NoriZC) [reply src/Accounts/Accounts/Context/GetAzureRMContext.cs:72]: We need to overwrite this virtual function so that context check can be skipped.

**Comment by NoriZC** [comment]
> Change log is required

This change is not expected to be seen by customer. If there will be other changes in Az.Accounts in this sprint, better not to leave this update in changelog

**Comment by BethanyZhou** [comment]
There is no change log for Az.Accounts by far

File: src/Accounts/Accounts/ChangeLog.md [hunk 2]
```diff
@@ -19,6 +19,14 @@
...
 
 ## Upcoming Release
+* Restored necessary initilization steps which were accidentally skipped for:
+    - `Disable-AzDataCollection`
+    - `Enable-AzDataCollection`
+    - `Remove-AzEnvironment`
+    - `Get-AzEnvironment`
+    - `Set-AzEnvironment`
+    - `Add-AzEnvironment`
+    - `Import-AzContext`
```
Comment (BethanyZhou) [inline src/Accounts/Accounts/ChangeLog.md]:
```suggestion
* Enabled telemetry collection for `Disable/Enable-AzDataCollection`, `Add/Get/Set/Remove-AzEnvironment` and `Import-AzContext`
```

**Review by BethanyZhou — APPROVED** [review]

**Pull request merged**